docs/vote-escrow/realms.mdx

```diff
@@ -157,7 +157,7 @@ then it declines linearly until release.
 
 ### Duration
 
-The duration of your cooldown is defined in days. The longer your tokens are staked, the more veHNT
+The duration of your cooldown is defined as days. The longer your tokens are staked, the more veHNT
```

Comment:
```suggestion
The duration of your cooldown is set in units of days. The longer your tokens are staked, the more veHNT
```
How about something like this?

Reply:
Too wordy imo, most of the people reading probably fly through anyways.